### PR TITLE
followup of #72364, slightly improve the comment

### PR DESCRIPTION
**Pull request by caesarxuchao**
See discussion [here](https://github.com/kubernetes/kubernetes/pull/72364#discussion_r246667887).

/sig api-machinery
/assign @kdada 
/kind cleanup
/release-note-none

### CODE REVIEW

**Comment by caesarxuchao** [comment]
@kdada I can't assign this to you since you are not a member, but could you review it (only the last commit, other commits will disappear after your #72364 lands)? I'll find some other member to give the lgtm label later. Thank you.

**Comment by kdada** [comment]
LGTM

**Comment by caesarxuchao** [comment]
/retest
/assign @cheftako 
to apply the label.

File: pkg/controller/garbagecollector/garbagecollector_test.go [hunk 1]
```diff
@@ -864,7 +864,12 @@ func TestGarbageCollectorSync(t *testing.T) {
 	//            gc.resyncMonitors()
 	//            controller.WaitForCacheSync() loops with `syncedPollPeriod` (hardcoded to 100ms), until either its stop channel is closed after `period`, or all caches synced.
 	//
-	// Setting the period to 200ms allows the WaitForCacheSync() to check for cache sync ~2 times in every wait.PollImmediateUntil() loop.
+	// Setting the period to 200ms allows the WaitForCacheSync() to check
+	// for cache sync ~2 times in every wait.PollImmediateUntil() loop.
```
Comment (cheftako) [inline pkg/controller/garbagecollector/garbagecollector_test.go]: Is this at least 2?

**Review by cheftako — APPROVED** [review]
/lgtm

File: pkg/controller/garbagecollector/garbagecollector_test.go (hunk 1 above)
Comment (caesarxuchao) [reply pkg/controller/garbagecollector/garbagecollector_test.go]: Unfortunately, this is really a "roughly 2" due to the nature of golang `select`.

**Pull request merged**